db/schema.rb

```diff
@@ -57,7 +57,6 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.index ["assignment_id"], name: "index_problems_on_assignment_id"
-    t.index ["solution_id"], name: "index_problems_on_solution_id"
```

Comment:
You need to rerun `make deploy` for the schema to be updated.

Simple way would be to revert this line manually